### DISCUSSION TITLE
How do I display the screenshots when the test has passed?

### DISCUSSION THREAD

**Discussion by brianbattformant** [allure-framework/allure2#2270] [Questions & Support]
When a test fails, I can click and see the screenshot.  However, when the test passes, I can't click on it:

<img width="438" alt="Screenshot 2023-12-28 at 9 37 53 AM" src="https://github.com/allure-framework/allure2/assets/153536845/7b99b936-a419-4d2b-9978-c135bb6aa452">

This is the command I'm using to generate the report:

`allure generate allure-results -o html-results --clean`

In the resulting folder, I can see the screenshots.  So, I know I'm attaching everything properly.  

<img width="542" alt="Screenshot 2023-12-28 at 9 41 47 AM" src="https://github.com/allure-framework/allure2/assets/153536845/83842cba-8d07-4d44-ad66-2caac8b77f20">

What do I need to do to make the screenshot accessible in the report?

**Comment by baev** [comment]
@brianbattformant please, specify the framework & the versions you're using
